doc/man3/ASN1_generate_nconf.pod

```diff
@@ -126,7 +126,7 @@ add EXPLICIT or IMPLICIT tagging, add wrappers or to change
 the string format of the final type and value. The supported
 formats are documented below.
 
-=over 2
+=over 4
```

Comment:
And examples *should* have a 1-or-more space indent, that's what makes them blocks of code.